core/src/main/java/org/apache/carbondata/core/datastore/filesystem/AbstractDFSCarbonFile.java

```diff
@@ -550,12 +550,10 @@ public DataOutputStream getDataOutputStreamUsingAppend(String path, FileFactory.
     if (null != fileStatus && fileStatus.isDirectory()) {
```

Comment:
Now the `pathFilter` is not used here? it may impact the callers of this method.

Reply:
Missed in rebase.. Resolving this now.